File: PJS5_POC_Cpp/Sources/Robot/Legs.cpp
#include "Legs.h"

namespace ROBOT
{
	LEGS::LEGS(const int iId, const std::string& sName, const int iArmor, const int iLegs)
		: IPARTS(iId, sName, iArmor, iLegs)
	{
	}

	LEGS::~LEGS()
	{

	}
}

File: PJS5_CSharp/PJS5_CSharp/Sources/Robot/Legs.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PJS5_CSharp.Sources.Robot
{
        public class LEGS : IPARTS
        {
            public LEGS(int iId, string sName, int iArmor, int iLegs) : base(iId, sName, iArmor, iLegs)
            {
            }

            ~LEGS()
            {
            }
        }
    }

